source/javascripts/directives/multi_select.js

```diff
@@ -0,0 +1,103 @@
+
```

Comment:
Since this is kind of a 3rd party library, shouldn't it go into the `vendor` directory?